components/datetime/Cargo.toml

```diff
@@ -22,27 +22,37 @@ include = [
 [dependencies]
 icu_locid = { version = "0.1", path = "../locid" }
 icu_provider = { version = "0.1", path = "../provider" }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
I'm planning to merge #451 today or tomorrow which makes a very similar change in this file